tests/pipeline/test_estimate.py

```
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import argparse
+import pandas as pd
+import sys
+import os
+import glob
+from typing import Optional
+import numpy as np
```

Ruff (0.11.9)

`numpy` imported but unused

Remove unused import: `numpy`

(F401)